distributed/client.py

```diff
@@ -5495,39 +5496,45 @@ async def __aexit__(self, exc_type, exc_value, traceback):
 class performance_report:
     """Gather performance report
 
-    This creates a static HTML file that includes many of the same plots of the
-    dashboard for later viewing.
+     This creates a static HTML file that includes many of the same plots of the
+     dashboard for later viewing.
 
-    The resulting file uses JavaScript, and so must be viewed with a web
-    browser.  Locally we recommend using ``python -m http.server`` or hosting
-    the file live online.
+     The resulting file uses JavaScript, and so must be viewed with a web
+     browser.  Locally we recommend using ``python -m http.server`` or hosting
+     the file live online.
```

Comment:
Why do we need to add an extra space here? Looking at the docs, it appears to be rendered correctly in `main`

Reply:
it was automatically added by black, when I run pre-commit hook. I'll remove them